documentation/source/conf.py

```diff
@@ -336,7 +336,7 @@
 # -- Extra setup for towncrier -------------------------------------------------
 # see also https://towncrier.readthedocs.io/en/actual-freaking-docs/
 
-in_progress_notes = subprocess.check_output(['towncrier', '--draft'],
+in_progress_notes = subprocess.check_output(['towncrier', '--draft', '--name', 'cocotb', '--version', release],
```

Comment:
```suggestion
# we pass the name and version directly, to avoid towncrier failing to import the non-installed version
in_progress_notes = subprocess.check_output(['towncrier', '--draft', '--name', 'cocotb', '--version', release],
```

Reply:
It seems to me a better solution would be to make towncrier use the installed version of cocotb rather than the source version. I guess that can come in a later PR if we can figure it out.

Reply:
It may be that towncrier does not need cocotb it just look at news fragments that are not part of cooctb, It just needs version and name. I am not an expert.